cql3/expr/restrictions.cc

```diff
@@ -193,6 +148,51 @@ void preliminary_binop_vaidation_checks(const binary_operator& binop) {
 }
 } // anonymous namespace
 
+void validate_single_column_relation(const column_value& lhs, oper_t oper, const schema& schema, bool is_lhs_subscripted) {
```

Comment:
Why did you need to move this function around in the file? It makes it impossible for me to check if you just moved it or changed anything.
If you did both (moving *and* changing), please separate it into two patches: One patch just moves the code (and the commit message says that it only moves the code), and then a patch that changes it.

Reply:
I had to move that function to move it out of namespace and place near another function declared in global namespace and restrictions.hh. To use `validate_single_column_relation` in `prepare_expr.cc`.
Okay, next time will (if possible) in separate commit. Either better always combine pull request's commits to the single one?